components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -161,13 +161,15 @@ def main(argv):
         "-t", "--tags", help="A comma-separated list of tags to apply to the compressed archives."
     )
     parsed_args = args_parser.parse_args(argv[1:])
+    compression_paths = parsed_args.paths
```

Comment:
Can we do some renaming:
* `--input-list` -> `--path-list`
* `compression_paths` -> `paths_arg` (or if you want to be more specific, `compress_paths_arg`)
* `compression_input_list` -> `path_list_arg` (or if you want to be more specific `compress_path_list_arg`)
* `list_path` -> `paths_to_compress`
* `InputConfig.list_path` -> `InputConfig.paths_to_compress`